src/components/Button/Follow.less

```diff
@@ -8,10 +8,13 @@
   .colorize-button(@purple, #2a3574);
 
   color: @white-darker;
+
+  & > .anticon {
+    margin-right: 12px;
+  }
 }
 
 .Follow--danger {
-  .btn;
-  .btn-default;
-  .btn-danger;
+  color: #fff !important;
+  .colorize-button(#f04134, #f04134);
```

Comment:
Can we have the same red than the one used on topic close? We can setup this color as default for `@red` aswell so it's easy to reuse.